deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -70,6 +70,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
                         <div class="form-group"></div>
                         <div class="form-group">
                             <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip"  title="Customize user data">User data configs:</label>
+                            <a href="https://w.pinadmin.com/pages/viewpage.action?spaceKey=IN&title=Launching+a+new+service#Launchinganewservice-DeploytheservicetoCMP/Dockercluster">wikiHelp</a>
```

Comment:
as it is open source, please add pinterest specific link to env variable, same as other .pinadmin.com url.

Reply:
Thanks for reminding, updated.